.github/workflows/continousint.yaml

```diff
@@ -15,4 +15,21 @@ jobs:
     steps:
     - uses: actions/checkout@v4
     - run: docker build --tag todo-app:test --target test .
-    - run: docker run todo-app:test   
\ No newline at end of file
+    - run: docker run todo-app:test
+    
+  deploy:
+    name: Push to Docker Hub and Trigger App Service for Deployment
+    runs-on: ubuntu-latest
+    needs: build
+    if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/Mod9CD' }} 
+    steps:
+    - uses: actions/checkout@v4
+    - run: echo hello deploy
```

Comment:
Good practice for clean code is to remove debug messages like this before merging.